core/domain/subtopic_page_domain.py

```diff
@@ -38,6 +41,8 @@
 # optionally, 'old_value'.
 CMD_UPDATE_SUBTOPIC_PAGE_PROPERTY = 'update_subtopic_page_property'
 
+WrittenTranslationsType = Dict[str, Dict[str, Dict[str, Dict[str, object]]]]
```

Comment:
Why the last `Dict` contains `object`?

Reply:
I did initially define it as `Dict[str, Dict[str, Dict[str, Dict[str, Union[str, bool]]]]]`. However, the mypy checks return an error that the expected type is `Dict[str, Dict[str, Dict[str, Dict[str, object]]]]`. Thus, I did so.

![Screenshot from 2022-02-18 16-51-46](https://user-images.githubusercontent.com/76519771/154673417-ecc24453-1e2b-476f-b7b7-148a7037f6b4.png)

Reply:
So `Dict[str, Dict[str, Dict[str, Dict[str, Union[str, bool]]]]]` or it does not work?

Reply:
No, `Dict[str, Dict[str, Dict[str, Dict[str, Union[str, bool]]]]]` still gives an error on the terminal.

Reply:
Can you try to force the `written_translations_dict: Dict[str, Dict[str, Dict[str, Dict[str, Union[str, bool]]]]] = {`?

Reply:
Ya, will do it!

Reply:
Any update on this?

Reply:
Sorry, had my midsems this week. Removing some tests you mentioned above led to issues in backend coverage. I'll fix them soon.